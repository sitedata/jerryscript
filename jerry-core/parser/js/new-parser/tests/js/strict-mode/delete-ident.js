/* This test contains strict mode parser tests. */

"use strict"

delete 1;
delete a();

a++;

delete a;
