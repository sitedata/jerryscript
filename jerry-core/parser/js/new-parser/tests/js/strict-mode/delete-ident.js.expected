
--- Script parsing start ---

  Note: switch to strict mode

  [  1] CBC_PUSH_LITERAL idx:0->string(use strict)
  [  0] CBC_POP_BLOCK
  [  1] CBC_PUSH_NUMBER_POS_BYTE number:1
  [  2] CBC_EXT_PUSH_UNDEFINED_BASE
  [  1] CBC_DELETE_PUSH_RESULT
  [  0] CBC_POP_BLOCK
  [  1] CBC_PUSH_LITERAL idx:1->ident(a)
  [  1] CBC_CALL0_PUSH_RESULT
  [  2] CBC_EXT_PUSH_UNDEFINED_BASE
  [  1] CBC_DELETE_PUSH_RESULT
  [  0] CBC_POP_BLOCK
  [  0] CBC_POST_INCR_IDENT_BLOCK idx:1->ident(a)

--- Script parsing end ---

Parse error 'Deleting identifier is not allowed in strict mode.' at line: 10 col: 9
